test/wasm2c/add.txt

```diff
@@ -0,0 +1,661 @@
+;;; TOOL: run-spec-wasm2c
+;;; ARGS: --print-output
+(module $adder
+	(func (export "add") (param i32 i32) (result i32)
+	      (i32.add (local.get 0) (local.get 1))))
+
+(register "adder" $adder)
+
+(assert_return (invoke $adder "add" (i32.const 3) (i32.const 4)) (i32.const 7))
```

Comment:
Is there any reason to include the `register` and `assert_return` line, if we are just interested in storing the output code?   i.e. Isn't the functionality of adding already covered by the spec tests?

Reply:
I had initially done the `register` so that `run-spec-wasm2c.py` would give the module a nicer name (e.g. `adder_instance_t` instead of `add_0_wasm_instance_t`) but now that we're using wasm2c directly this doesn't matter -- removed.